src/Formplot/Common/DoubleCompareExtensions.cs

```diff
@@ -24,18 +24,18 @@ namespace Zeiss.PiWeb.Formplot.Common
 	/// https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/using-nullable-types
 	///
 	/// Comparing nullables returns:
```

Comment:
Wenn die Testabdeckung in diesem Projekt wichtig ist, sollten die entsprechenden Tests (siehe `DoubleComopareExtensionsTest`) dupliziert werden.